templates/components/argument.js.sapphire

```diff
@@ -1,15 +1,23 @@
 { "category": "arguments" }
 ---
-import { Argument } from "@sapphire/framework";
+const { Argument } = require('@sapphire/framework');
```

Comment:
Shouldn't we keep using imports/exports and suggest ESM for users? :3
Would be better for future proofing

Reply:
1. That is a very very very distant future
2. That would be a separate template at such time as `argument.esm.sapphire`